src/helperV1.js

```diff
@@ -29,7 +29,7 @@ export async function signSendChunkv1(app, chunkIdx, chunkNum, chunk) {
     payloadType = PAYLOAD_TYPE.LAST;
   }
   return app.transport
-    .send(CLA, INS.SIGN_ED25519, payloadType, 0, chunk, [0x9000, 0x6984, 0x6a80])
+    .send(CLA, ins, payloadType, 0, chunk, [0x9000, 0x6984, 0x6a80])
```

Comment:
is this meant to be a new parameter like in the changes below?

Reply:
Thanks for noticing @pro-wh ! It is fixed now!